morph_tool/transform.py

```diff
@@ -234,10 +233,12 @@ def align_morphology(
         L.info('We did not find an apical point to align the morphology')
         return np.eye(3)
 
-    principal_direction = _get_principal_direction(points)
+    center = np.mean(morph.soma.points, axis=0)
```

Comment:
@arnaudon I just realized: shouldn't we use the soma center here?
`center = morph.soma.center`
Because the soma center has different definitions depending on the soma type.

Reply:
Ah probably, I never know this soma thing, feel free to change that!

Reply:
Ah ok, I thought the method was not the same for all soma types. I guess it would still be better to use the `morph.soma.center` property to be sure that this center is computed the same way as in other codes. I will create a PR for that.